### DISCUSSION TITLE
用 AI 抓八成人頭帳戶，北富銀怎做到「被詐騙集團除名」

### DISCUSSION THREAD

**Discussion by RavenJiaSin** [1101-GS4719-C/Python-Community-GS4719-C#1715] [General]
台灣的詐騙越發猖獗，據警政署統計，去年詐騙案達29,700多件，總財損金額逾69億6千萬元，年增23%，創下新高。涉案人頭帳戶會被檢調單位列為「警示戶」，金錢不得進出。北富銀則將防線向前推，利用AI模型提前發現可疑帳戶，依法進行控管。截至今年上半年，全台警示戶累計破10萬戶，眾家銀行警示戶不斷增加， 唯獨北富銀占全體銀行警示戶比率減少近14%。這代表，詐騙集團的人頭帳戶難以在此存活。許多詐騙集團甚至會在收購人頭帳戶的小廣告上，直接標註「不收北富銀帳戶」。

以往當民眾臨櫃匯錢，要匯入的帳戶雖非警示戶，但行員認為可疑，這時，只能苦言相勸並報警處理，因為如果自行凍結該帳戶，可能會遭客訴。即便警察到場，由於當事人沒有報案，未出現受害人、也沒有被害事實，警方當下也只能兩手一攤，等對方發現上當，來報案才能受理，但往往為時已晚。而這個問題的關鍵在於：缺乏有力佐證。，如果對當事人所用的帳戶，能合理懷疑是人頭帳戶，早一步將該帳戶暫時禁用，就有機會為被害人爭取一些時間，免於金錢損失。

於是北富銀的團隊與與刑事局合作，通過整理警示帳戶與客戶交易紀錄的特徵，讓AI學習，建構出模型。同時要與時俱進，不停納入新的詐騙帳戶交易態樣，才能持續因應詐騙新手法。目前該AI模組揪出詐騙集團人頭帳戶的準確率可高達八成。

有了強大的AI模型判斷可疑帳戶，還需要搭配現場資深行員進行應對與處裡。正所謂「在AI時代，真正的價值還是人！」，北富銀在金融安全的邏輯上，思考布局和機制都到位，才能成事。AI只是工具，如果沒有資深行員的經驗值，也無法提升準確度，櫃台也仍需要真人同事的機警應對。

然而僅有一家銀行被詐騙集團除名並沒有辦法打擊到詐騙生態，因此北富銀決定將AI模型無償分享給其他銀行，讓整個金融體系，共同建置起防護網。然而銀行同業是彼此競爭的關係，要組織起聯盟、拿出資料分享，談何容易？負責與聯盟成員溝通的北富銀金融安全部襄理劉書夢說，各家銀行剛開始行事謹慎，會詢問很多問題，甚至在北富銀要求各家銀行提供相關數據資料的時候，也頻頻受質疑。因此北富銀的團隊在會議上，率先開誠布公、不藏私的分享，而且有問必答，提供他們在內部與各單位溝通時的有利依據，才逐漸建立互信。金融安全部副總經理蔡佩玲表示：「今天在國家傷口上，大家不用競爭，你要加入我們，（所有資料）就是透明，這是最高指導原則。」

原文出處：[科技新報 用 AI 抓八成人頭帳戶，北富銀怎做到「被詐騙集團除名」](https://technews.tw/2023/11/27/taipei-fubon-used-ai-to-catch-80percent-of-dummy-accounts/)

從以上我們可以得到兩個重點：
首先是AI技術與人的合作能產生更大的價值。在北富銀的案例中我們可以發現該AI模型起到了為臨櫃行員提供分析異常帳號可能性的證明，文章中打擊詐欺犯罪中心警務正張佳勳說銀行阻詐機制成功的兩大要件為「AI模型辨識人頭帳戶的精準度高，以及銀行行員願意強制執行凍結帳戶，」
第二點是我們可以看到AI技術的共享以及在行業競爭的環境中如何合作以達成共利的目標，我認為不只是在金融界，若技術能被分享，那將會為社會帶來更大的利益。如同文章中北富銀風控長蔡永原將其以疫苗做比喻：「這就像是疫苗，我們先產生了第一代的疫苗，有用，讓別人也可以使用這項專利。因為每家銀行的客戶結構不一樣，各家可以再進一步發展出不同疫苗，就有機會一起往前走」

**Comment by mrweixd** [comment]
利用AI做到Security的部分確實是個優良的點子。
